src/api/app/views/webui2/webui/package/_deps.html.haml

```diff
@@ -0,0 +1,59 @@
+.row
+  .table-responsive.col-sm-12.col-md-6
+    %h3 Provides
+    %table.table.table-hover
+      %thead.thead-light
+        %tr
+          %th.wr-50{ scope: 'col' } Symbol
+          %th.wr-50{ scope: 'col' } Required by
```

Comment:
Let me know if you know a better way to set the width of the columns (ensuring that the second column is not getting squeezed when it's empty).
Not sure if `wr-50` is the right way to do this here.

Reply:
It might be possible with the `col-*` classes [here](https://getbootstrap.com/docs/4.1/layout/grid/)